shell/imports/client/signup-client.js

```diff
@@ -41,7 +52,7 @@ Router.map(function () {
     },
 
     data: function () {
-      const keyInfo = globalDb.collections.signupKeys.findOne(this.params.key);
+      const keyInfo = SandstormDb.collections.signupKeys.findOne(this.params.key);
```

Comment:
One more spot you missed, also in this file. I think this is the last one. (Also, to head it off, I suspect once you fix this the linter will tell you that `SandstormDb` is an unused import

Reply:
Happy to. One thing though.  This is not in a template helper, it's in a route.  I'm not at all sure that Template.instance() works here, and I'm also concerned that if it doesn't it won't show up in my testing.

Do you have a suggestion?

Reply:
We could just punt and keep using globalDb here for now.

Reply:
I spent 20 minutes trying to find out how the "data" variable gets set on a template, but I was unsuccessful.  I'm taking your advice and punting.

Pushed the changes.